packages/client/src/components/ExportButton.tsx

```diff
@@ -98,9 +123,55 @@ const ExportButton = () => {
 		}
 	};
 
+	const customTheme: CustomFlowbiteTheme = {
+		alert: {
+			color: {
+				red: 'bg-red-100 text-red-900',
+			},
+		},
+	};
+
 	return (
 		<>
 			<Script src="https://accounts.google.com/gsi/client" async></Script>
+			<div className="flex flex-col space-y-2 mb-8">
+				<Dropdown
+					color="gray"
+					label={
+						transcriptFormat === null
+							? 'Choose transcript format'
+							: transcriptFormatDescription[transcriptFormat]
+					}
+				>
+					<Dropdown.Item
+						value={TranscriptFormat.TEXT}
+						onClick={() => {
+							setTranscriptFormat(TranscriptFormat.TEXT);
+							setTranscriptFormatValid(true);
+						}}
+					>
+						{transcriptFormatDescription[TranscriptFormat.TEXT]}
+					</Dropdown.Item>
+					<Dropdown.Divider />
+					<Dropdown.Item
```

Comment:
Should we make srt the default option in the dropdown? From journalist feedback it seems like this is the format they want most of the time.

Reply:
This is more of a UX point. I think by enforcing to choose one of the formats we are also letting user know that there are multiple formats to choose from, but if there's a format chosen by default, user might not even notice there are other formats. We can chat with Luke and see what he says too.

Reply:
I think Text might still be the most popular format - I suspect if we set SRT as the default initially then everyone would have emailed asking for a timecode-free version

Reply:
good points! let's go with no default